plugins/win-capture/window-helpers.c

```diff
@@ -383,6 +383,64 @@ static int window_rating(HWND window, enum window_priority priority,
 	return val;
 }
 
+static int window_rating_by_list(HWND window, const DARRAY(struct game_capture_picking_info) * games_whitelist)
```

Comment:
The name of the function is a bit confusing to me, would `is_game_window` for example be more descriptive of what the function does?

Additionally, I'm having trouble to understand why this function couldn't simply return a boolean.

Reply:
With functions "find_window_one_of()" and "window_rating_by_list()" I tried to keep naming similar to name used for mode with exact window selected "find_window()" and "window_rating()".

Return value is not boolean as it hard to detect exact window. Title of window for example can include a version of a game and that require to make fuzy comparison and select best matching window.